libraries/AP_Scheduler/AP_Scheduler.h

```diff
@@ -93,6 +94,10 @@ class AP_Scheduler
     uint32_t get_loop_period_us() const {
         return 1000000UL / _loop_rate_hz;
     }
+    // get the time-allowed-per-loop:
+    float get_loop_period_s() const {
+        return 1.0 / _loop_rate_hz;
```

Comment:
This is getting called a lot. It would be more computationally efficient if we checked it slower, like 1 Hz, and cached the result. Same goes for get_loop_period_us()

Reply:
There's discussion in this PR about this very issue.  Please read and add your comments.